Sources/Unbox.swift

```diff
@@ -692,6 +692,46 @@ public class Unboxer {
         })
     }
     
+    /// Unbox a required Array containing values that can be formatted using a formatter (optionally allowing invalid elements)
+    public func unbox<T: UnboxableWithFormatter, F: UnboxFormatter where F.UnboxFormattedType == T>(key: String, isKeyPath: Bool = true, formatter: F, allowInvalidElements: Bool = false) -> [T] {
+        return UnboxValueResolver<[F.UnboxRawValueType]>(self).resolveRequiredValueForKey(key, isKeyPath: isKeyPath, fallbackValue: [], transform: { (array) -> [T]? in
+            if allowInvalidElements {
+                return array.flatMap({ formatter.formatUnboxedValue($0) })
+            } else {
+                return array.map({ (value) -> T in
+                    if let formattedValue = formatter.formatUnboxedValue(value) {
+                        return formattedValue
+                    }
+                    
+                    self.failForInvalidValue(value, forKey: key)
+                    return T.unboxFallbackValue()
+                })
+            }
+        })
+    }
+    
+    /// Unbox an optional Array containing values that can be formatted using a formatter (optionally allowing invalid elements)
+    public func unbox<T: UnboxableWithFormatter, F: UnboxFormatter where F.UnboxFormattedType == T>(key: String, isKeyPath: Bool = true, formatter: F, allowInvalidElements: Bool = false) -> [T]? {
+        return UnboxValueResolver<[F.UnboxRawValueType]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath, transform: { (array) -> [T]? in
+            if allowInvalidElements {
+                return array.flatMap({ formatter.formatUnboxedValue($0) })
+            } else {
+                var invalidElement = false
+                
+                let mapping = array.flatMap({ (value) -> T? in
+                    if let formattedValue = formatter.formatUnboxedValue(value) {
+                        return formattedValue
+                    }
+                    
+                    invalidElement = true
```

Comment:
Why continue with the loop if an invalid element was found? Rather than using a mapping function, wouldn't it be better with a classic `for .. in` loop that you can just `return nil` from if an invalid value was found?